packages/nodes-base/nodes/Gitlab/GitlabTrigger.node.ts

```diff
@@ -175,11 +194,11 @@ export class GitlabTrigger implements INodeType {
 					events[`${e}_events`] = true;
 				}
 
-				const endpoint = `/projects/${owner}%2F${repository}/hooks`;
+				const endpoint = `/projects/${projectId}/hooks`;
 
 				const body = {
 					url: webhookUrl,
-					events,
+					...events,
```

Comment:
This is not sufficient to fix #663, "push_events" needs to be explicitly set to false. I couldn't find anything about `push_events` being implicitly true if not set in the Gitlab documentation.

I suggest that the request should include all known fields set to an explicit boolean value.

If I add `events['push_events'] = false;` after line 187 it works as expected.

Reply:
Ahh yes, thanks for pointing that out.